src/index.js

```diff
@@ -1,102 +1,3 @@
-require('dotenv').config();
```

Comment:
Why remove this?

Put this back...... invoking this only once at the `index.js` make lot of sense